widget/embedded/src/components/FilterSelector/FilterSelector.style.ts

```diff
@@ -0,0 +1,92 @@
+import { css, darkTheme, IconButton, styled } from '@rango-dev/ui';
+
+export const IconContainer = styled('div', {
+  position: 'relative',
+  '&::before': {
+    position: 'absolute',
+    right: '1px',
+    top: '-1px',
+    width: '$8',
+    height: '$8',
+    borderRadius: '100%',
+    backgroundColor: '$neutral300',
+    [`.${darkTheme} &`]: {
+      backgroundColor: '$neutral400',
+    },
+    padding: '$2',
+  },
+  variants: {
+    isSelect: {
+      true: {
+        '&::before': {
+          content: '',
+        },
+      },
+    },
+  },
+});
+
+export const FilterButton = styled(IconButton, {
+  width: '$36',
+  height: '$36',
+  position: 'relative',
+  padding: '0',
+  overflow: 'unset',
+  backgroundColor: '$neutral300',
+  [`.${darkTheme} &`]: {
+    backgroundColor: '$neutral400',
+  },
+  '&:hover': {
+    backgroundColor: '$secondary100',
+    [`.${darkTheme} &`]: {
+      backgroundColor: '$neutral',
+    },
+    [`& ${IconContainer}::before`]: {
+      backgroundColor: '$secondary100',
+      [`.${darkTheme} &`]: {
+        backgroundColor: '$neutral',
+      },
+    },
+  },
+  variants: {
+    isSelect: {
+      true: {
+        border: '1px solid $secondary',
+      },
+    },
+  },
+});
+
+export const Badge = styled('div', {
+  position: 'absolute',
+  width: '$6',
+  height: '$6',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  borderRadius: '3px',
+  top: '$0',
+  right: '$2',
```

Comment:
One pixel is a better match with Figma.
```suggestion
  right: '1px',
```

Reply:
Fixed